src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLSPOptions.cs

```diff
@@ -40,15 +27,23 @@ public RazorLSPOptions(bool enableFormatting, bool autoClosingTags, bool commitE
                                                          FormatOnType: true,
                                                          AutoInsertAttributeQuotes: true,
                                                          ColorBackground: false,
+                                                         CodeBlockBraceOnNextLine: false,
                                                          CommitElementsWithSpace: true);
 
     /// <summary>
     /// Initializes the LSP options with the settings from the passed in client settings, and default values for anything
     /// not defined in client settings.
     /// </summary>
-    internal static RazorLSPOptions From(ClientSettings clientSettings)
+    internal static RazorLSPOptions From(ClientSettings settings)
```

Comment:
good catch :)

Reply:
I don't think there were any issues here, it just seemed like way too much code :)

Reply:
A shorter name that's equally understandable is always a good change in my book.

I've spent many a commit reducing `projectSnapshotManagerDispatcher` to `dispatcher`. 😁

Reply:
oh, I meant that you removed the ctor and updated the `From`, which seemed to be missing some things

Reply:
or rather, easier to miss things. I think as we added more things the move to getting rid of the ctor is a good call.

Reply:
I don't think it was missing anything, it was just confusing. The code essentially just moved from the constructor to the From method, since that was the only caller. I actually maybe should have made From into a constructor 🤷‍♂️